packages/components/src/FormGroup/FormGroup.tsx

```diff
@@ -104,4 +108,8 @@ function FormGroup({
 FormGroup.Title = Title;
 FormGroup.Items = Items;
 
+FormGroup.displayName = "FormGroup";
```

Comment:
We have to set the `displayName` for the `FormGroup` itself in order for the docstring to continue to appear in storybook.